tests/test_hodgkin.py

```diff
@@ -136,7 +136,9 @@ def test_state_symbols(self, model):
         _potassium_channel_n_gate$n
         """
         state_symbols = model.get_state_symbols()
-        assert len(state_symbols) == 4
+        state_symbols_ordered_by_input = model.get_state_symbols(order_by_order_added=True)
+        assert len(state_symbols) == len(state_symbols_ordered_by_input) == 4
+        assert set(state_symbols) == set(state_symbols_ordered_by_input)
```

Comment:
Not really a good test since the order is the same for the ordered and unordered cases.

Reply:
Yes well the original also wasn't a good test if you're honest since it doesn't check the state symbols it just counts them, so I changed that one too.